torch/testing/_internal/common_modules.py

```diff
@@ -2990,7 +2990,9 @@ def module_error_inputs_torch_nn_Pad3d(module_info, device, dtype, requires_grad
                skips=(
                    DecorateInfo(unittest.skip("Skipped!"), 'TestModule', 'test_grad'),
                    DecorateInfo(unittest.skip("Skipped!"), 'TestModule', 'test_gradgrad'),
-                   DecorateInfo(skipIfMps, 'TestModule', dtypes=[torch.float64]),)
+                   DecorateInfo(skipIfMps, 'TestModule', dtypes=[torch.float64]),
+                   # Fails with channels last test on MPS backend
+                   DecorateInfo(unittest.expectedFailure, "TestModule", "test_memory_format", device_type='mps'),)
```

Comment:
Why is this new skip introduced? This used to work and doesn't anymore?

Reply:
No, it's not a regression. This test always failed. This skip is already in place from another PR:
https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_modules.py#L3069 

So I'll remove this change from my PR.